lib/image.ts

```diff
@@ -0,0 +1,92 @@
+import * as _ from 'lodash';
+import { fs } from 'mz';
```

Comment:
Eventually it'll be possible to just use
```ts
import { promises as fs } from 'fs'
```
but we still need this for now due to node 6/8 :(